openpype/hosts/maya/plugins/load/load_abc_to_standin.py

```diff
@@ -0,0 +1,114 @@
+import os
+
+from openpype.pipeline import (
+    load,
+    get_representation_path
+)
+from openpype.settings import get_project_settings
+
+
+class AlembicStandinLoader(load.LoaderPlugin):
+    """Load Alembic as Arnold Standin"""
+
+    families = ["model", "pointcache"]
+    representations = ["abc"]
+
+    label = "Import Alembic as Standin"
+    order = -5
+    icon = "code-fork"
+    color = "orange"
+
+    def load(self, context, name, namespace, options):
+
+        import maya.cmds as cmds
+        import pymel.core as pm
```

Comment:
With issue https://github.com/pypeclub/OpenPype/issues/2353 open I'd recommend avoiding Pymel where possible and rewriting this using maya cmds

Reply:
I agree. I also noticed another ArnoldStandin Loader would have an error message of `Error: Problem with the API object returned by __apiobject__ method` when loading ass file under some circumstance. I guess it's related to the pymel command too.